### ISSUE TITLE
Crouch gait

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7297]
**Preferred term label:**
Crouch gait

**Definition (free text, please give PubMed ID)**
Gait disturbance which is characterized by excessive ankle dorsiflexion, knee and hip flexion during the stance phase. 
(PMID: 26709688; 28279852)

**Parent term (use hpo.jax.org/app)**
Gait disturbance HP:0001288

**Comment**
It is a gait deviation, observed in some of the patients affected by cerebral palsy, which increases energy costs of walking and contributes to ambulatory decline over time.

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
added, thanks @jagcsun
